src/store/actions/authentication.ts

```diff
@@ -38,8 +39,8 @@ export const prepHomePage = (dispatch) => {
 
 export const checkCredentials = ({ email, password }) => {
     return async (dispatch) => {
-        // let dummyId = 1;
-        // let profile = null;
+        let dummyId = 1;
```

Comment:
this dummyId seems to change, why is it instantiated as 1?

Reply:
cause we needed a dummy id to see if it worked?

Reply:
i mean it doesn't have to be I'll change that if you want